models/user.js

```diff
@@ -1,18 +1,14 @@
-var Base = require('./base');
-var Book = require('./book');
+const bookshelf = require('../db/bookshelf');
 
-
-
-module.exports = Base.extend({
+module.exports = bookshelf.Model.extend({
```

Comment:
Is this ok that you changed this from Base.extend to bookshelf.Model.extend?

Reply:
Woops, I think I just left that while debugging!